<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pattaya Budget (c) PMC</string>
  </property>
  <widget class="QLabel" name="days_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Days</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="days">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLabel" name="ladys_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ladys</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ladys">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="hotel_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hotel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="hotel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>450</number>
   </property>
  </widget>
  <widget class="QLabel" name="food_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Food</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="food">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bars">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="bars_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bars / Booze</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_Label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TOTAL</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_baht_Label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Baht</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_euro_Label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Euro</string>
   </property>
  </widget>
  <widget class="QLabel" name="exchange_Label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exchange Rate</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="exchange">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>42.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="sum_euro_Label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_baht_Label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update!</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="taxi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="taxi_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Taxi</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="airlineTicket">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>850</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Airline ticket eur</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="monthlySavings">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QLabel" name="monthlySavings_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Monthly Savings</string>
   </property>
  </widget>
  <widget class="QLabel" name="monthsToSave_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Months to save</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_months_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Funds</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="currentFunds">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
